### ISSUE TITLE
Am I missing something?

### ISSUE THREAD

**Issue by Togezx** [Developer-Mike/Minecraft-Bedrock-Formation-Finder-1.18#5]
Hey, I'm trying to find a base with this but it doesn't seem to be working properly. I used a random formation of bedrock to test it and the coordinates I was given were in a completely different place. I checked for typos as well but nothing seems to be working. The only command I've found that works is one that looks for 3x3x3 cubes of bedrock. Anyone know a solution? Here's a screenshot of the formation I'm using for testing: https://imgur.com/FN1WLdk

**Comment by Developer-Mike** [comment]
That's strange. I'd be happy if someone contributes a fix for it because I am no longer familiar with this code and have other priorities.

**Comment by JanCantCode** [comment]
> Hey, I'm trying to find a base with this but it doesn't seem to be working properly. I used a random formation of bedrock to test it and the coordinates I was given were in a completely different place. I checked for typos as well but nothing seems to be working. The only command I've found that works is one that looks for 3x3x3 cubes of bedrock. Anyone know a solution? Here's a screenshot of the formation I'm using for testing: https://imgur.com/FN1WLdk

the issue here is that the given bedrock pattern is not being "rotated". this means if your pattern is not perfectly symmetrical, whilst there will be locations found with the pattern, not all locations that have this pattern will be found. this is not too much work in order to fix, which i am currently working on

**Comment by Developer-Mike** [comment]
You're right! You can make a pull request if you've successfully implemented it 👍🏻

**Comment by JanCantCode** [comment]
will do so (i technically have it working but its not as nice as i want it)

Mike. ***@***.***> schrieb am So., 7. Jan. 2024, 08:30:
